kpops/components/common/streams_bootstrap.py

```diff
@@ -72,3 +73,11 @@ def warning_for_latest_image_tag(self) -> Self:
                 f"The image tag for component '{self.name}' is set or defaulted to 'latest'. Please, consider providing a stable image tag."
             )
         return self
+
+    @override
+    async def reset(self, dry_run: bool) -> None:
+        await self.destroy(dry_run)
+
+    @override
+    async def clean(self, dry_run: bool) -> None:
+        await self.destroy(dry_run)
```

Comment:
should we provide this as default implementation for `PipelineComponent` or are there any drawbacks?

Reply:
I was considering doing so, and I think it is the correct way of handling `reset/clean.` Suppose you initiate a `clean` on your pipeline and have a `HelmApp` component. If we don't call it `destroy` in the `PipelineComponent` class, the component will not be uninstalled.